### PR TITLE
Bugfix: Checkpoint fix datetime parsing problem

### PR DESCRIPTION
**Pull request by vg-svitla**

### CODE REVIEW

File: Checkpoint/client/http_client.py [hunk 1]
```diff
@@ -81,20 +81,60 @@ async def get_harmony_mobile_alerts(
...
+        Also, sometimes it can be `No data` value.
+
+        Args:
+            value: str
+
+        Returns:
+            datetime | None:
+        """
+        if value is None:
+            return None
```
Comment (squioc) [inline Checkpoint/client/http_client.py:128]: Should we not add a test about `No data` and return `None` in this case?

**Review by squioc — APPROVED** [review]
